packages/backend/src/tools/Logger.ts

```diff
@@ -22,6 +22,7 @@ export class Logger {
   constructor(private options: LoggerOptions) {}
 
   static SILENT = new Logger({ logLevel: LogLevel.NONE, format: 'pretty' })
+  static DEBUG = new Logger({ logLevel: LogLevel.DEBUG, format: 'pretty' })
```

Comment:
I'm not 100% sure it's needed here, but it was very useful for testing. I can move it to `test` directory if you'd like @sz-piotr.

Reply:
I think it's a nice change.